tests/array/array.cpp

```diff
@@ -58,6 +58,8 @@ int main(const int argc, const char *argv[]) {
   rb[7] = 0.0;
   a->copy_out(rb);
   AERO_ASSERT(rb[7] == 12.3);
+  AERO_ASSERT(a->data()[7] == 12.3);
```

Comment:
Same comment goes for these floating point comparisons (and the Fortran ones, while we're at it).